README.md

```diff
@@ -1,2 +1,16 @@
-# display-url-api
-Provides the DisplayURLProvider extension point to provide alternate URLs for use in notifications
+# Display URL API Plugin for Jenkins
+[![Jenkins Plugins](https://img.shields.io/jenkins/plugin/v/display-url-api)](https://plugins.jenkins.io/display-url-api)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/display-url-api?color=blue)](https://plugins.jenkins.io/display-url-api)
+
+Provides an extension point to define alternate URLs for use in notifications.
+
+URLs can be requested/extended for these UI locations:
+
+-   Root page.
+-   Job.
+-   Run.
+-   Run changes.
+-   Test result.
+
+##  Version history
+Please refer to the [changelog](CHANGELOG.md)
```

Comment:
I didn't realize relative links worked correctly from the main repo page, good to know!